system/Database/Postgre/Builder.php

```diff
@@ -312,6 +313,105 @@ public function join(string $table, $cond, string $type = '', ?bool $escape = nu
         return parent::join($table, $cond, $type, $escape);
     }
 
+    /**
+     * Generates a platform-specific batch update string from the supplied data
+     *
+     * @used-by batchExecute
+     *
+     * @param string                          $table  Protected table name
+     * @param list<string>                    $keys   QBKeys
+     * @param list<list<int|SqlValue|string>> $values QBSet
+     */
+    protected function _updateBatch(string $table, array $keys, array $values): string
+    {
+        $sql = $this->QBOptions['sql'] ?? '';
+
+        // if this is the first iteration of batch then we need to build skeleton sql
+        if ($sql === '') {
+            $constraints = $this->QBOptions['constraints'] ?? [];
+
+            if ($constraints === []) {
+                if ($this->db->DBDebug) {
+                    throw new DatabaseException('You must specify a constraint to match on for batch updates.'); // @codeCoverageIgnore
+                }
+
+                return ''; // @codeCoverageIgnore
+            }
+
+            $updateFields = $this->QBOptions['updateFields'] ??
+                $this->updateFields($keys, false, $constraints)->QBOptions['updateFields'] ??
+                [];
+
+            $alias = $this->QBOptions['alias'] ?? '_u';
```

Comment:
Alias name is not getting quoted like:

"alias"."column"

Reply:
It seems `$this->QBOptions['alias']` is quoted.
https://github.com/codeigniter4/CodeIgniter4/blob/0f43ba12bd4636af7c2332ae6026ddf9be4402e9/system/Database/BaseBuilder.php#L2010